docs/user-manual/civiform-admin-guide/using-markdown.md

```diff
@@ -0,0 +1,25 @@
+# Using Markdown
+
+Markdown is a simple formatting tool that allows you to create text links and lists as well as apply bold, italics, font sizes, and other formatting edits to text. CiviForm Admin can use Markdown to format text in four key places within CiviForm:
+- Program descriptions
+- Custom confirmation screen messages
+- Static question text
+- Regular question text (Note: For regular question text, please limit formatting to bold, italics and text links only. Changing font size or making other edits that alter the flow of text can make question text hard to read)
+
+The formatting options currently supported by CiviForm are outlined in the table below along with how to use them. If you would like a formatting option that we do not currently support, let us know!
+
+![Markdown table 1](../../.gitbook/assets/markdown.png)
```

Comment:
The only thing I'm a little waffling on is the examples as an image because people can't copy/paste then edit it. They're such simple examples though so it's probably fine.

Reply:
I'm going to leave it as an image for now because I think it's easier to visually parse, but I agree copy/paste would be nice. I'll change it if folks ask for that.